test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -162,7 +162,7 @@ describe('33acrossBidAdapter:', function () {
   function ServerRequestBuilder() {
     const serverRequest = {
       'method': 'POST',
-      'url': END_POINT,
+      'url': `${END_POINT}?guid=${SITE_ID}`,
```

Comment:
No spec to test passing guid in url param?

Reply:
i think there is no need for a specific test for this. because there are already multiple tests case checking the full url that included the guid. also there are already a test case for checking the ttxsetting url when its given.

Reply:
Oh right the spec would check against the above `serverRequest` results object which now includes GUID, got it.